layers/deft/packages.el

```diff
@@ -37,4 +37,9 @@
         "d" 'deft-delete-file
         "i" 'deft-toggle-incremental-search
         "n" 'deft-new-file
-        "r" 'deft-rename-file))))
+        "r" 'deft-rename-file)
+      (evil-define-key 'insert deft-mode-map
+        (kbd "C-h") 'deft-filter-clear
+        (kbd "C-j") 'evil-next-line
```

Comment:
Hum, I just thought that this will not work for non-evil users :-( Not event sure `C-k` should be remapped as discussed in https://github.com/syl20bnr/spacemacs/issues/5237.